crates/starknet-devnet-core/src/starknet/predeployed.rs

```diff
@@ -4,26 +4,23 @@ use starknet_types::contract_storage_key::ContractStorageKey;
 use starknet_types::felt::Felt;
 
 use crate::constants::{
-    CHARGEABLE_ACCOUNT_ADDRESS, ERC20_CONTRACT_CLASS_HASH, ERC20_CONTRACT_PATH,
-    UDC_CONTRACT_ADDRESS, UDC_CONTRACT_CLASS_HASH, UDC_CONTRACT_PATH,
+    CHARGEABLE_ACCOUNT_ADDRESS, UDC_CONTRACT_ADDRESS, UDC_CONTRACT_CLASS_HASH, UDC_CONTRACT_PATH,
 };
 use crate::error::{DevnetResult, Error};
 use crate::state::StarknetState;
 use crate::system_contract::SystemContract;
 use crate::traits::StateChanger;
 use crate::utils::get_storage_var_address;
 
-pub(crate) fn create_erc20_at_address(contract_address: &str) -> DevnetResult<SystemContract> {
-    let erc20_contract_class_json_str =
-        std::fs::read_to_string(ERC20_CONTRACT_PATH).map_err(|err| Error::ReadFileError {
-            source: err,
-            path: ERC20_CONTRACT_PATH.to_string(),
-        })?;
-    let erc20_fee_contract = SystemContract::new(
-        ERC20_CONTRACT_CLASS_HASH,
-        contract_address,
-        &erc20_contract_class_json_str,
-    )?;
+pub(crate) fn create_erc20_at_address(
+    contract_address: &str,
+    path: &str,
+    hash: &str,
```

Comment:
Why should this now be specified if in production we will always use the same cairo 1 implementation? Shouldn't the path and the hash be hardcoded?

Reply:
Thanks for the review. It's a remnant of my refactor where I was mixing both versions.

Fixing...

Reply:
Fixed in 94bd3f331a5ae81dadf01b0cf9efe8a451c29bfd